app/models/concerns/metadata_location_helpers.rb

```diff
@@ -0,0 +1,32 @@
+module MetadataLocationHelpers
+  extend ActiveSupport::Concern
+
+  included do
+    attr_accessor :location_ids
```

Comment:
We may don't need this custom `MetadataLocationHelpers` because we could just use

```rb
class User
  store_accessor :metadata,
                 :location_ids
end
```

Then, you just use `User#location_ids` as it's like an attribute under User.

Reply:
@samnang, i put it here because i think we could reuse for contact too. So, do you think we should have it separately for each model?